src/lib/operators/maintenance/create_table.hpp

```diff
@@ -1,29 +1,38 @@
 #pragma once
 
-#include "operators/abstract_read_only_operator.hpp"
+#include "operators/abstract_read_write_operator.hpp"
+#include "operators/insert.hpp"
 #include "storage/table_column_definition.hpp"
 
 namespace opossum {
 
 // maintenance operator for the "CREATE TABLE" sql statement
-class CreateTable : public AbstractReadOnlyOperator {
+class CreateTable : public AbstractReadWriteOperator {
```

Comment:
Following from what @mrks said below, this can probably stay a AbstractReadOnlyOperator. It should force the internal Insert operator to initialize the begin commit ids to 0 - probably via a dummy TransactionContext

Reply:
Mmh, I'd rather have proper commit ids here. After all, those are used to identify a snapshot of the database. It's bad enough that we cannot deal with schema modifications, but we should not allow rows to suddenly become visible to already running transactions.

Reply:
Okay